client/utils/SkillsList.ts

```diff
@@ -0,0 +1,27 @@
+export const skillList = [
```

Comment:
Should put this in `client/constants.ts` or `client/constants/skills.ts` or something like that since it's not really utils-related.

Reply:
Done